src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -13,6 +13,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         VisualBasic12 = 12
         VisualBasic14 = 14
         VisualBasic15 = 15
+        VBvNext = Integer.MaxValue
```

Comment:
They are under a /feature flag. but that requires a non-null `ParseOptions` so that they can be consistently and correctly check for.